src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -201,6 +202,10 @@ protected final void mainEntry(final String[] args) {
             }
             handleUserException(e);
             System.exit(COMMANDLINE_EXCEPTION_EXIT_VALUE);
+        } catch (final PicardNonZeroExitException e) {
+            // a Picard tool returned a non-zero exit code
+            handleResult(e.getToolReturnCode());
+            System.exit(ANY_OTHER_EXCEPTION_EXIT_VALUE);
```

Comment:
I think it would be better to have a separate exit status code for the case where a Picard tool exits with non-0, so that clients will know to check the log to get the underlying Picard exit code.

Reply:
Done.